tv.plex.Plexamp.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: tv.plex.Plexamp
+
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
```

Comment:
current runtime is 22.08. 21.08 will be EOL in less than 3 month.

Reply:
Our app currently doesn't launch with the newer runtime. A future release has significantly upgraded dependencies, and I'm expecting this issue to be resolved in the timeframe of 4-6 weeks.